src/olympia/devhub/views.py

```diff
@@ -1481,14 +1484,20 @@ def submit_addon_upload(request, step, channel):
 @submit_step(4)
 def submit_describe(request, addon_id, addon, step):
     forms_, context = [], {}
-    describe_form = forms.DescribeForm(request.POST or None, instance=addon,
-                                       request=request)
+    # Figure out the latest version early in order to pass the same instance to
+    # each form that needs it (otherwise they might overwrite each other).
+    latest_version = addon.find_latest_version(
+        channel=amo.RELEASE_CHANNEL_LISTED)
```

Comment:
Ok. I think it's better to prevent them from using this view entirely, otherwise we'll spend our time chasing edge cases...